apps/full-site-editing/package.json

```diff
@@ -27,14 +27,14 @@
 		"build": "rimraf dist && npm-run-all --parallel build:* && npm run copy-to-dist"
 	},
 	"dependencies": {
-		"@wordpress/blocks": "^6.2.3",
-		"@wordpress/components": "^7.3.0",
-		"@wordpress/compose": "^3.2.0",
-		"@wordpress/data": "^4.4.0",
-		"@wordpress/editor": "^9.2.4",
-		"@wordpress/element": "^2.3.0",
-		"@wordpress/i18n": "^3.3.0",
-		"classnames": "^2.2.6",
-		"lodash": "^4.17.11"
+		"@wordpress/blocks": "6.0.7",
```

Comment:
This looked wrong to me, but then I was surprised to find that `@wordpress/blocks` really is locked to v6.0.7 in the `npm-shrinkwrap.json`: https://github.com/Automattic/wp-calypso/blob/ecaeee16da13d04ad09bbe2fd8844590196256c7/npm-shrinkwrap.json#L3605-L3631

Does anyone know how that's possible? I don't know if it's a bug or a feature, and if it's a shrinkwrap thing specifically or not.

Reply:
nicely spotted @rarkins. We're still working out how we're going to use the monorepo for these kinds of things and how that works out with npm and internal file refs..

Reply:
@blowery right now that above table looks pretty funny though, e.g.

![image](https://user-images.githubusercontent.com/6311784/57080272-7c037e80-6cf2-11e9-9489-da26bfb56680.png)

I guess.. that's another reason to pin deps -  you can be sure what you're getting! But I do not understand how the lock file ended up mismatching the package.json like that.